appvm-scripts/usrbin/qubes-session

```diff
@@ -50,25 +51,6 @@ if [ -f /etc/X11/Xsession.d/90qubes-keymap ]; then
 	. /etc/X11/Xsession.d/90qubes-keymap
 fi
 
-# Save user environment
-(
-    umask 0077
-    unset PWD TERM SHELL RUNLEVEL PATH SHLVL LOGNAME USER MEM
-    export -p > /tmp/qubes-session-env.tmp
-)
-
-cd $HOME
-
-if [ -x /usr/bin/qrexec-fork-server ]; then
-    /usr/bin/qrexec-fork-server
-fi
-
-# Environment ready, make it available to processes
-mv /tmp/qubes-session-env.tmp /tmp/qubes-session-env
-
-# Wake waiting for environment processes
-[ -s /tmp/qubes-session-waiter ] && kill -KILL `cat /tmp/qubes-session-waiter`
-
```

Comment:
I'm very happy to remove this completely, but we need to be absolutely sure this change won't break anything.
Ideas of what to check:
 - starting GUI application from a menu
 - calling a service between VMs, that results in a GUI (like qvm-open-in-vm)
 - both the above to a DisposableVM
 - calling a service to a VM with non-default user (root?) that potentially shows a window (I think `qubes-dom0-update --gui` does it)
 - compare environment before and after this change and see if looks sane (`qvm-run -p VMNAME env`, `qvm-run -u root -p VMNAME env`)
 - verify that nothing else check existence of `/tmp/qubes-session-env`

Related issues to look on (this could make it either better or worse):
https://github.com/QubesOS/qubes-issues/issues/4976
https://github.com/QubesOS/qubes-issues/issues/3533

Reply:
Ok I will do the tests